framework/include/transfers/MultiAppTransfer.h

```diff
@@ -38,7 +38,7 @@ class MultiAppTransfer : public Transfer
   /**
    * Utility to verify that the variable in the destination system exists.
    */
-  void variableIntegrityCheck(const AuxVariableName & var_name) const;
+  void variableIntegrityCheck(const AuxVariableName & var_name, bool is_from_multiapp) const;
```

Comment:
This appears to have broken SAM: https://civet.inl.gov/job/2718600/

Reply:
the failure I observed, SAM not building, was not related
https://civet.inl.gov/job/2715195/

Reply:
This is a simple fix. I ll make a patch now

Reply:
Patch there
https://git-nse.egs.anl.gov/SAM/SAM/-/merge_requests/1023